requirements.txt

```diff
@@ -1,9 +1,12 @@
 # Evennia dependencies
 
 # general
+attrs >= 19.2.0
 django >= 2.2.5, < 2.3
 twisted >= 19.2.1, < 20.0.0
 pytz
+djangorestframework >= 3.10.3
```

Comment:
Would it not be safer to restrict these to an upper version number as well?

Reply:
Any preference between `< 3.11` or `< 4.0`?

Reply:
I don't know offhand what DRF's version policy is and if 3.11 would be a backwards-incompatible version or that won't happen until 4.0 (if they have such considerations).

Reply:
Turns out that I had 3.11 installed, so that looks to be safe. I'll do `< 3.12`